CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing to HyperUI
+
+Thank you for your interest in improving HyperUI. Please read through this and the supporting blog post [Contributing to HyperUI](https://www.hyperui.dev/blog/how-to-contribute) carefully.
+
+## Getting Started
+```bash
+git clone git@github.com:markmead/hyperui.git
+yarn install
+yarn dev
+yarn css --watch
+```
+
+ℹ️ **Note**
+
+Please ignore the console warning ```params should be awaited before using its properties``` as this is not relevant.
```

Comment:
This isn't correct, it should be using GitHub markdown callouts - https://github.com/orgs/community/discussions/16925